ios/Classes/TiMapModule.h

```diff
@@ -32,6 +32,9 @@ return [NSNumber numberWithInt:map];\
 @property(nonatomic,readonly) NSNumber *HYBRID_FLYOVER_TYPE;
 @property(nonatomic,readonly) NSNumber *SATELLITE_FLYOVER_TYPE;
 #endif
+#ifdef __IPHONE_11_0
```

Comment:
Is this the correct way to check it? I was asking the same to me and ended up using:
```objc
#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 110000
  // Write iOS 11 code here
#endif
```
The reason I've read about is because Apple recommends to use the enum-values instead of the macros mapped to them, so you would always have a value to compare without relying on it's define-state. Please also apply for other parts of this app or define an IS_IOS_11 like we done with IS_XCODE_9 in timob.

Reply:
created IS_IOS_11 similar to IS_XCODE_9 in TiMAp_Prefix.pch .